kxml_compiler/classdescription.cpp

```diff
@@ -80,11 +80,8 @@ ClassProperty::List ClassDescription::properties() const
 
 bool ClassDescription::hasProperty(const QString &name) const
 {
-    foreach (ClassProperty p, m_properties) {
-        if (p.name() == name)
-            return true;
-    }
-    return false;
+    return (std::any_of(m_properties.begin(), m_properties.end(),
```

Comment:
`return` isn't a function, I prefer `return foo;` over `return (foo);`

i.e. I suggest to remove the extra () around everything.